BunqSdk/Exception/BadRequestException.cs

```diff
@@ -2,7 +2,8 @@
 {
     public class BadRequestException : ApiException
     {
-        public BadRequestException(int responseCode, string message) : base(responseCode, message)
+        public BadRequestException(int responseCode, string message, string responseId) :
+            base(responseCode, message, responseId)
```

Comment:
I'd choose to move the `:` to the new line (like [here](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/using-constructors))